packages/rainbowkit/src/components/Dialog/DialogContent.css.ts

```diff
@@ -42,14 +42,8 @@ export const dialogContentWideMobile = style([
 export const dialogContentWideDesktop = style([
   dialogContent,
   {
-    width: largeScreenMinWidth,
-  },
-  {
-    '@media': {
-      [`screen and (min-width: ${largeScreenMinWidth}px)`]: {
-        width: '720px',
-      },
-    },
+    minWidth: '720px',
+    width: '720px',
```

Comment:
Should be very similar to how we do with [compact modal style](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/components/Dialog/DialogContent.css.ts#L58-L61)